functions.inc.php

```diff
@@ -654,7 +654,26 @@ function list_domains () {
     return $list;
 }
 
+//
+// list_servers
+// Action: List all available servers.
+// Call: list_servers ()
+//
+function list_servers () {
+    $list = array();
 
+    $table_server = table_by_key('server');
+    $list[null] = "none";
+    $result = db_query ("SELECT server FROM $table_server ORDER BY server");
```

Comment:
Please use ServerHandler->getList() instead of "raw" SQL queries - see list_admins() for an example.

Since you only call this function in PFAHandler, you might as well embed the 3 lines of code there - functions.inc.php is already big enough ;-)